amazon-clone/src/App.vue

```diff
@@ -1,15 +1,19 @@
 <template>
-  <navbar-component />
   
+  <navbar-component />
+ <router-view></router-view>
+
 </template>
 
 <script>
 import NavbarComponent from "./components/layout/NavbarComponent.vue";
+//import RegistrationForm from "./components/RegistrationForm.vue"
```

Comment:
remove this

Reply:
ok Sir

Reply:
Done in my remote repo please approve this now